From df795e92869655f3fe4ce66d619bc521e16cf633 Mon Sep 17 00:00:00 2001
From: Yurii Zubrytskyi <zyy@google.com>
Date: Wed, 22 Apr 2020 15:39:32 -0700
Subject: [PATCH 3/6] [cleanup] Fix a clang-tidy warning

Move constructors should be noexcept where possible

Bug: 153704006
Test: builds & boots
Change-Id: I9de0b09a6b1e29cb9c2e21cdf05b96983fa6091c
---
 libs/binder/include/binder/ParcelFileDescriptor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/binder/include/binder/ParcelFileDescriptor.h b/libs/binder/include/binder/ParcelFileDescriptor.h
index 4635ad84c6..072e1e99d3 100644
--- a/libs/binder/include/binder/ParcelFileDescriptor.h
+++ b/libs/binder/include/binder/ParcelFileDescriptor.h
@@ -31,7 +31,7 @@ class ParcelFileDescriptor : public android::Parcelable {
 public:
     ParcelFileDescriptor();
     explicit ParcelFileDescriptor(android::base::unique_fd fd);
-    ParcelFileDescriptor(ParcelFileDescriptor&& other) : mFd(std::move(other.mFd)) { }
+    ParcelFileDescriptor(ParcelFileDescriptor&& other) noexcept : mFd(std::move(other.mFd)) { }
     ~ParcelFileDescriptor() override;
 
     int get() const { return mFd.get(); }
-- 
2.20.1

